app_helper/utils.py

```diff
@@ -16,6 +15,11 @@
 
 from . import HELPER_FILE
 
+try:
+    from distutils.version import LooseVersion
+except ImportError:  # pragma: no cover
+    from setuptools import LooseVersion
+
```

Comment:
just a code style note: I would keep the updated version in the `try` and the legacy one in the except clause, it communicates best what's the "right" thing to do

Same for lines 70-78

```suggestion
try:
    from setuptools import LooseVersion
except ImportError:  # pragma: no cover
    from distutils.version import LooseVersion
```